mininet/wifiMobility.py

```diff
@@ -180,7 +180,8 @@ def associate_wpa(self, sta, ap, wlan):
         :param wlan: wlan ID
         """
         passwd = self.verifyPasswd(sta, ap, wlan)
-        os.system('pkill -f \'wpa_supplicant -B -Dnl80211 -i %s\'' % sta.params['wlan'][wlan])
+        confname = "mn%d_%s_wifiDirect.conf" % (os.getpid(), sta.name)
```

Comment:
I guess wpa_supplicant uses the given namespace, when started with `sta.cmd(...)`. Would it be possible to change `os.system('pkill ...')` to `sta.cmd('pkill ...')`? I think this would "kill two birds with one stone".